src/main/scala/org/broadinstitute/dsde/workbench/leonardo/api/LeoRoutes.scala

```diff
@@ -42,6 +42,13 @@ class LeoRoutes(val leonardoService: LeonardoService, val proxyService: ProxySer
             }
           }
         }
+      } ~
+      get {
+        complete {
+          leonardoService.getClusterDetails(googleProject, clusterName). map { clusterDetails =>
+            StatusCodes.OK -> clusterDetails
```

Comment:
I think `OK` is set by default for successful Futures, so would just need to do:
```
complete {
  leonardoService.getClusterDetails(googleProject, clusterName)
}
```

Reply:
i think you're right, but leaving it explicit makes people more likely to a) write it themselves b) think about whether it should be `OK` vs. e.g. `Created`

Reply:
I vote for clarity - should we change the put endpoint to`Created`?

Reply:
PUT should definitely be 201, yeah